### PR TITLE
Covered the article "Ember Simple Auth Video"

### PR DESCRIPTION
**Pull request by fozy81**
## Ember Times review template

- [x] Add your name to top and bottom
- [x] Put emoji in writeup title
- [x] Add a short blurb (could be the title) to the beginning
- [x] Link to external article/repo/etc in paragraph/body text, not just the writeup title (link in writeup title is optional now)
- [x] Add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account
- [x] Check that all links work

### CODE REVIEW

File: content/the-ember-times-issue-180.md [hunk 1]
```diff
@@ -10,19 +10,17 @@ tags:
...
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+## [1. Ember Simple Auth Video 📽](https://www.youtube.com/watch?v=bSWN4_EbTPI)
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+Need a friendly, step by step guide to adding authentication to your Ember app? Look no further than the Ember Simple Auth [Quickstart video](https://www.youtube.com/watch?v=bSWN4_EbTPI) published by [Simplabs](https://simplabs.com/). Acting as a helpful introduction to authentication/authorization using this popular [Ember addon](https://ember-simple-auth.com/).
```
Comment (MinThaMie) [inline content/the-ember-times-issue-180.md]:
```suggestion
Are you looking for a friendly, step by step guide to adding authentication to your Ember app? Look no further than the Ember Simple Auth [Quickstart video](https://www.youtube.com/watch?v=bSWN4_EbTPI) published by [Simplabs](https://simplabs.com/). Acting as a helpful introduction to authentication/authorization using this popular [Ember addon](https://ember-simple-auth.com/).
```

Comment (MinThaMie) [reply content/the-ember-times-issue-180.md]: I was thinking that maybe you can give a bit more context about this video. For example, include that Marco is the one explaining it. And maybe if you found this yourself superhelpfull, maybe you can point towards what made it valuable for you :)

**Comment by fozy81** [comment]
@MinThaMie Thanks for feedback, I've updated with a little more context. Please review, thanks.

File: content/the-ember-times-issue-180.md [hunk 2]
```diff
@@ -10,19 +10,17 @@ tags:
...
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+## [1. Ember Simple Auth Video 📽](https://www.youtube.com/watch?v=bSWN4_EbTPI)
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+Are you looking for a friendly, step by step guide to adding authentication to your Ember app? Look no further than the Ember Simple Auth [Quickstart video](https://www.youtube.com/watch?v=bSWN4_EbTPI) published by [Simplabs](https://simplabs.com/). This video provides helpful introduction to this popular authentication/authorization [Ember addon](https://ember-simple-auth.com/). [Marco](https://twitter.com/marcoow) explains this tricky part of app development using the latest Ember Octane syntax.
```
Comment (MinThaMie) [inline content/the-ember-times-issue-180.md]:
```suggestion
Are you looking for a friendly, step by step guide to adding authentication to your Ember app? Look no further than the Ember Simple Auth [Quickstart video](https://www.youtube.com/watch?v=bSWN4_EbTPI) published by [Simplabs](https://simplabs.com/). This video provides helpful introduction to this popular authentication/authorization [Ember addon](https://ember-simple-auth.com/). [Marco Otte-Witte (@marcoow)](https://github.com/marcoow) explains this tricky part of app development using the latest Ember Octane syntax.
```

**Review by MinThaMie — APPROVED** [review]
You did a great job! Thanks for your first write-up 🥳

File: content/the-ember-times-issue-180.md [hunk 3]
```diff
@@ -10,19 +10,17 @@ tags:
...
 ---
 
-## [1. Section title in sentence case 🐹](section-url)
+<!--alex disable simple-->
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+## [1. Ember Simple Auth Video 📽](https://www.youtube.com/watch?v=bSWN4_EbTPI)
```
Comment (amyrlam) [inline content/the-ember-times-issue-180.md]:
```suggestion
## [Ember Simple Auth Video 📽](https://www.youtube.com/watch?v=bSWN4_EbTPI)
```

**Review by amyrlam — APPROVED** [review]
Thank you @fozy81, much appreciated, looks great! 🙌

**Comment by amyrlam** [comment]
Feel free to merge in, or I can later! (Sometimes I go ahead and merge them in if we are close to publish day / they've been reviewed)

**Review by fozy81 — COMMENTED** [review]
Thanks for catching that.

**Pull request merged**